articles/libraries/lock-ios/v2/touchid-authentication.md

```diff
@@ -0,0 +1,68 @@
+---
+section: libraries
+toc_title: Touch ID Authentication
+description: How to implement Touch ID authentication with Lock iOS.
+---
+# Lock iOS: Touch ID Authentication
+
+Here's the scenario: You are using `webAuth` to present the HLP (Hosted Login page) for the user to Login. After user authentication you want to store the user's credentials and use the `refreshToken` to renew the user's credentials without having to present the HLP. Instead you want to use Touch ID to authenticate and renew the user's credentials.
```

Comment:
This makes some noise. "After user authentication you want to store the user's credentials and use the `refreshToken` to renew the user's credentials without having to present the HLP."

It seems like we're assuming that the user instead of using the saved credentials, the user will always renew them before use. We should be saying something like "After user authentication, you want to store the credentials and use them as long as they are valid. Once they expire, you would want to renew them using the `refreshToken` in order to avoid presenting the HLP again."   instead.......